source/ODApplication.cpp

```diff
@@ -87,8 +92,6 @@ void ODApplication::startServer()
 void ODApplication::startClient()
 {
     ResourceManager& resMgr = ResourceManager::getSingleton();
-    LogManagerOgre logManager(resMgr.getLogFile());
-    logManager.setLogDetail(LogMessageLevel::NORMAL);
```

Comment:
"startGame" calls both "startServer" and "startClient". Both implementations can now use the same logging mechanism, using the sinks. I added a LogSinkOgre as well, but only realized after the fact that it writes to a similar log file.